packages/katello/rubygem-katello/rubygem-katello.spec

```diff
@@ -340,5 +347,8 @@ cp -pa .%{gem_dir}/* \
 %{gem_instdir}/webpack
 
 %changelog
+* Tue Jul 17 2018 Eric D. Helms <ericdhelms@gmail.com> 3.8.0-0.1.rc1
+- rebuilt
```

Comment:
Please fix this here and in other places :)

Reply:
Is the indentation off?